examples/PlayVGM/decompressVgz.bat

```diff
@@ -0,0 +1,6 @@
+REM This script will decompress all *.vgz-files in a directory to *.vgm
+REM 
+REM Note: Requires gzip.exe (http://gnuwin32.sourceforge.net)
+
+copy *.vgz *.vgm.gz
+gzip -d *.vgm.gz
```

Comment:
Would it be possible for you to also add a `.sh` file to decompress *.vgz files for Linux or Mac users?

Reply:
done, tested in windows with git bash...